linux_sql/README.md

```diff
@@ -1,2 +1,90 @@
 # Linux Cluster Monitoring Agent
-This project is under devel opment. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Review.
+
+## Introduction
+This cluster monitoring agent is used to record the hardware specifications of each node and
+ monitor node resource usage in real-time. The collected data can be used to generate report
+s by the infrastructure team which helps them to make better resource planning in the future
+.
+
+## Architecture & Design
+
+![diagram](./assets/architecture.png)
+
```

Comment:
use a title here to make it more readable 

e.g.
`### Tables`

Reply:
Thanks. Updated!